src/applications/find-forms/components/FindVaForms.jsx

```diff
@@ -26,6 +26,9 @@ export default function() {
           >
             <span>
               Read more
+              <span className="vads-u-visibility--screen-reader">
+                about filing a VA disability claim
+              </span>
```

Comment:
This should make the SR read `Read more about filing a VA disability claim`

Reply:
Heh, I've been using `sr-only` which is defined in the `packages/formation/sass/base/_va.scss` file. That one has a "newer" definition. I like it!

Reply:
I just discovered this myself!